correios/models/posting.py

```diff
@@ -524,7 +524,7 @@ def __init__(self,
         self.invoice_number = invoice_number
         self.invoice_series = invoice_series
         self.invoice_type = invoice_type
-        self.value = value
+        self._value = value
```

Comment:
to mantain consistency with other properties I suggest you to name this attribute as `real_value`